tests/PropertyList.py

```diff
@@ -102,14 +102,14 @@ def testExists(self):
 
     def testGetVector(self):
         apl = dafBase.PropertyList()
-        v = (42, 2008, 1)
+        v = [42, 2008, 1]
         apl.setInt("ints", v)
-        apl.setInt("ints2", (10, 9, 8))
+        apl.setInt("ints2", [10, 9, 8])
```

Comment:
Yes it is. This is one of the points where we really carefully need to think what we want. I see two options:

1. Consistently use lists for things that are `list` or `vector` in C++ and `tuple` for `pair` and `tuple`. Which is what I did now.
2. Change pybind11 to accept any sequence type.

Note that the second option can still lead to some confusion if people pass in a tuple but get back a list. Swig behaves quite differently here because it uses objects that behave as lists (or tuples) but are actually views into the underlying C++ containers. Personally I am leaning strongly towards not introducing these views in the short time, but do implement option 2 in the long term. Just not now...